crates/bevy_tasks/src/single_threaded_task_pool.rs

```diff
@@ -106,10 +106,12 @@ impl TaskPool {
         T: Send + 'static,
     {
         let executor = &async_executor::LocalExecutor::new();
+        // TODO: Safety comment
```

Comment:
Copying the safety comment from the multithreaded version is fine.

Reply:
Got it! Fixed in 1e2bd8f2101efc396dcadc14514b19aef6d79ebe.